Aphrodis/MyDiary_Challenge
# Ch footer #169517949

## What does this PR do?
Add footer to all UI pages and reduce the number of colors used on each page

## Description of tasks to be completed?
- Add footers to UI pages
- Use the same colour in all UI pages

**How should this be manually tested?**
In your terminal or command line:
- Clone the repo using: ```git clone https://github.com/Aphrodis/MyDiary_Challenge.git```
- Enter in the folder by doing ```cd MyDiary_Challenge```
- Check out the branch by ```git checkout ch-footer-#169517949```
- Run UI pages in the browser
**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
#169517949

**Screenshots**
![1](https://user-images.githubusercontent.com/34691089/68073278-12deef80-fd97-11e9-9a8f-6565d6a93258.PNG)
![2](https://user-images.githubusercontent.com/34691089/68073280-12deef80-fd97-11e9-9145-79975044a6c3.PNG)
![3](https://user-images.githubusercontent.com/34691089/68073281-13778600-fd97-11e9-9b11-37cded1fb0d8.PNG)
![4](https://user-images.githubusercontent.com/34691089/68073282-13778600-fd97-11e9-842b-fce633036e78.PNG)
![5](https://user-images.githubusercontent.com/34691089/68073283-14101c80-fd97-11e9-9840-e78d6a737d0d.PNG)
![6](https://user-images.githubusercontent.com/34691089/68073284-14101c80-fd97-11e9-82bd-2b6f9a144e5a.PNG)
